src/vs/workbench/contrib/debug/browser/debugHover.ts

```diff
@@ -116,10 +118,10 @@ export class DebugHoverWidget implements IContentWidget {
 		this.valueContainer = $('.value');
 		this.valueContainer.tabIndex = 0;
 		this.valueContainer.setAttribute('role', 'tooltip');
+		this.tip.innerText = 'Quick tip: Hold alt key to switch to editor hover';
```

Comment:
I would prefer the following text:
'Hold alt key to switch to editor language hover'

Reply:
And make sure to call `nls.localize` on the string so we get it translated in different lagnuages